config-overrides.js

```diff
@@ -0,0 +1,38 @@
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const path = require('path');
+
+function extractNamespaceAndLocale(absoluteFilename) {
+  const normalizedPath = path.normalize(absoluteFilename);
+  const localesIndex = normalizedPath.indexOf('/locales/');
+
+  const componentPath = normalizedPath.slice(0, localesIndex);
+  const namespace = componentPath.split(/[\\/]/).pop();
+
+  const localePath = normalizedPath.slice(localesIndex + '/locales/'.length);
+  const locale = localePath.split(/[\\/]/)[0];
+
+  return { namespace, locale };
+}
+
+module.exports = function override(config) {
+  if (!config.plugins) {
+    config.plugins = [];
+  }
+
+  config.plugins.push(
+    new CopyWebpackPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, 'src/**/locales/*/strings.json'),
+          to: ({ absoluteFilename }) => {
+            const { namespace, locale } = extractNamespaceAndLocale(absoluteFilename);
+            return `component-locales/${namespace}/${locale}/strings.json`
```

Comment:
A small yet known caveat of this implementation: 
Localization for a duplicated-component-name bound to override each other during build. Willing to take the chance for it to happen since:
a: Components aren't usually supposed to have the same names, I wouldn't recommend it to happen (import confusion, etc).
b: Time constraints ^.^"
This could be handled if I were to include the full absolute path, I opted against it.